### PR TITLE
Component ownership changes for Organizations and Locations and Logging

### PR DESCRIPTION
**Pull request by shweta83**

### CODE REVIEW

**Comment by omkarkhatavkar** [comment]
Can one of the admins verify this patch?

**Comment by mshriver** [comment]
@shweta83 Please rebase instead of merging, as this creates merge commits and pollutes the commit history.

There is a conflicting file as well, which you should be able to resolve through a rebase.

Don't hesitate to reach out internally if you'd like some guidance on your local git workflow :)

File: docs/index.rst [hunk 1]
```diff
@@ -25,89 +25,101 @@ ensure that Paramiko's dependencies build correctly:
 * Python development headers
 * libffi
 
-On Fedora, you can install these with the following command:
 
-For python2.x::
```
Comment (mshriver) [inline docs/index.rst:30]: Were you intending to change this file?

Comment (shweta83) [reply docs/index.rst:30]: This is rebasing issue. Will fix this.

**Pull request closed without merging**